package.json

```diff
@@ -1,32 +1,19 @@
 {
-  "name": "@datadog/native-iast-rewriter",
-  "homepage": "https://github.com/DataDog/dd-native-iast-rewriter-js/blob/main/README.md",
-  "version": "2.8.0",
-  "description": "Datadog IAST instrumentation addon for NodeJS",
+  "name": "@datadog/wasm-js-rewriter",
+  "homepage": "https://github.com/DataDog/dd-wasm-js-rewriter/blob/main/README.md",
+  "version": "3.0.0",
+  "description": "Datadog instrumentation addon for NodeJS",
   "main": "main.js",
   "types": "index.d.ts",
-  "napi": {
-    "name": "iast-rewriter",
-    "triples": {
-      "defaults": false,
-      "additional": [
-        "x86_64-unknown-linux-gnu",
-        "x86_64-unknown-linux-musl",
-        "x86_64-pc-windows-msvc",
-        "aarch64-apple-darwin",
-        "x86_64-apple-darwin"
-      ]
-    }
-  },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/DataDog/dd-native-iast-rewriter-js.git"
+    "url": "git+https://github.com/DataDog/dd-wasm-js-rewriter.git"
   },
   "keywords": ["datadog", "iast"],
```

Comment:
should we replace iast by rewriter? or at least add `rewriter` to the list?

Reply:
replaced